docs/src/constrained/irregular_surrogates.md

```diff
@@ -0,0 +1,25 @@
+# Irregular Timeseries Surrogates
```

Comment:
Capitalization only for the first word. I'd also say "Surrogates for time series with irregular time indices", or something along those lines, to be specific about what is irregular.